policy.md

```diff
@@ -1,181 +1,122 @@
+CCADB uses an Audit Letter Validation (ALV) tool to automatically parse and validate audit statements. If the audit statement fails to meet any of the following requirements, the CA Owner must work with their auditor to provide an audit statement that passes ALV.
 
-Audit statements listed in the CCADB must contain at least the following clearly-labelled information in English:
+Audit statements listed in the CCADB must contain at least the following clearly-labeled information in English:
 
 1. Name and address of the organization performing the audit;
-1. Full name of the CA that was audited;
-1. SHA-256 fingerprint of each root and intermediate certificate that was in scope of the audit (see format specifications below);
-1. List of the CA policy documents (with version numbers) referenced during the audit;
-1. Whether the audit is for a period of time or a point in time;
-1. Date the audit statement was written, which will necessarily be after the audit period end date or point-in-time date (see date format specifications below);
-1. Start date and end date of the period that was audited, for those that cover a period of time (this is not the period the auditor was on-site);
-1. Point-in-time date, for those that are for a point in time;
-1. Full names and version numbers of the audit standards that were used during the audit; and
-1. For ETSI, a statement to indicate if the audit was a full audit, and which parts of the criteria were applied, e.g. DVCP, OVCP, NCP, NCP+, LCP, EVCP, EVCP+, QCP-w, Part1 (General Requirements), and/or Part 2 (Requirements for trust service providers).
+2. Full name of the CA that was audited;
+3. SHA-256 fingerprint of each root and subordinate CA certificate that was in scope of the audit (see format specifications below);
+4. List of the CA policy documents (with version numbers) referenced during the audit;
+5. Whether the audit is for a period of time or a point in time;
+6. Date the audit statement was written, which will necessarily be after the audit period end date or point-in-time date (see date format specifications below);
+7. Start date and end date of the period that was audited, for those that cover a period of time (this is not the period the auditor was on-site);
+8. Point-in-time date, for those that are for a point in time;
+9. Full names and version numbers of the audit standards that were used during the audit; 
+10. All incidents disclosed by the CA Owner, discovered by an auditor, or reported by a third party, that, at any time during the audit period, occurred, were open in Bugzilla, or were reported to a Store; and
```

Comment:
I think this statement would read better as: `All incidents disclosed by the CA Owner, or reported by a third party, and all findings discovered by an auditor, that, at any time during the audit period, occurred, were open in Bugzilla, or were reported to a Store; and`

Audits create findings (qualification for WebTrust and non-conformity for ETSI), and a finding should result in an incident (in accordance with this proposed policy update).